kubernetes.tf

```diff
@@ -64,12 +64,18 @@ resource "aws_iam_openid_connect_provider" "quortex_cluster" {
   url             = aws_eks_cluster.quortex.identity[0].oidc[0].issuer
 }
 
+
 # Worker nodes
 
 resource "aws_launch_template" "quortex" {
+  count = var.force_imdsv2_on_managed_nodegroups ? 1 : 0
+
   name = aws_eks_cluster.quortex.name
   metadata_options {
-    http_tokens = "required"
+    http_endpoint               = "enabled"
+    http_tokens                 = "required"
+    http_put_response_hop_limit = 1
+    instance_metadata_tags      = "disabled"
```

Comment:
Why did all this was added ?

Reply:
Don't ask why, values are mostly default values, but ain't working without.
    instance_metadata_tags      = "disabled" is important though, or it'll break and ASG won't be able to schedule new nodes because of tags. Looks a lot like a bug on AWS side to me. Anyway.